public/jsx/components/product-detail.jsx

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var React = require('react');
+var products = require('products');
+var gettext = require('utils').gettext;
+
+module.exports = React.createClass({
+
+  displayName: 'ProductDetail',
+
+  propTypes: {
+    productId: React.PropTypes.string.isRequired,
+  },
+
+  render: function() {
+
+    var productId = this.props.productId;
+    var productData = products[productId];
+
+    if (!productData) {
+      throw new Error('Invalid product: ' + productId);
```

Comment:
I think we should show a user friendly error for this: https://github.com/mozilla/payments-ui/issues/99